libcore/Interfaces/SidebarInterface.vala

```diff
@@ -0,0 +1,51 @@
+/***
+    Copyright (c) 2020 elementary LLC <https://elementary.io>
```

Comment:
```suggestion
    Copyright 2020 elementary, Inc <https://elementary.io>
```

Reply:
Done - Need a separate PR to fix all the existing headers.